Code/Tools/LevelEdit/TGAToDXT.cpp

```diff
@@ -125,7 +125,7 @@ bool TGAToDXTClass::Convert (const char *inputpathname, const char *outputpathna
 				}
 
 				if (!redundantalpha) {
-
+					errorcode = 0;
```

Comment:
This warning was introduced by us having no replacement for nvDXT.
So perhaps we should emit a warning/error instead, and do `errorCode = -1;` so the function returns `false`?